deploy/kubernetes/helm/che/custom-charts/che-plugin-registry/templates/deployment.yaml

```diff
@@ -0,0 +1,61 @@
+#
+# Copyright (c) 2018 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: che
+    component: plugin-registry
+  name: plugin-registry
+spec:
+  replicas: 1
+  revisionHistoryLimit: 2
+  selector:
+    matchLabels:
+      app: che
+      component: plugin-registry
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 25%
+      maxUnavailable: 25%
+  template:
+    metadata:
+      labels:
+        app: che
+        component: plugin-registry
+    spec:
+      containers:
+      - image: {{ .Values.chePluginRegistry.image }}
+        imagePullPolicy: {{ .Values.chePluginRegistry.imagePullPolicy }}
+        name: che-plugin-registry
+        ports:
+        - containerPort: 8080
+        livenessProbe:
+          httpGet:
+            path: /plugins/
```

Comment:
AFAIK /plugins is deprecated so should we wait for another path like v3/plugins ?

Reply:
yes, fixed

Reply:
PR for plugin-registry https://github.com/eclipse/che-plugin-registry/pull/189